docs/spark-procedures.md

```diff
@@ -227,6 +228,12 @@ Remove snapshots older than specific day and time, but retain the last 100 snaps
 CALL hive_prod.system.expire_snapshots('db.sample', TIMESTAMP '2021-06-30 00:00:00.000', 100)
 ```
 
+Remove snapshots with snapshot ID `123` (note that this snapshot ID should not be the current snapshot):
```

Comment:
a) it might be good to mention more than one snapshot id here. 
b) Do we also mention what happens if those snapshots are part of branch or tag?